plugins/pagerduty/plugin.spec.yaml

```diff
@@ -68,6 +85,183 @@ types:
       - owner
       - read_only_user
       - user
+  incident_output:
+    id:
+      description: the id of the incident
+      type: string
+      example: PT4KHLK
+    type:
+      description: A string that determines the schema of the object
+      type: string
+      example: incident
+    summary:
+      description: A short-form, server-generated string that provides succinct, important information about an object suitable for primary labeling of an entity in a client. In many cases, this will be identical to name, though it is not intended to be an identifier
+      type: string
+      example: "[#1234] The server is on fire."
+    self:
+      description: the API show URL at which the object is accessible
+      type: string
+      example: https://api.pagerduty.com/incidents/PT4KHLK
```

Comment:
🔴

Reply:
All of these examples were fake data or else taken from the pager duty api docs, but i'll update them to make them easier to look fake